### ISSUE TITLE
[Improvements] Check for updates on the gbfs feed database

### ISSUE THREAD

**Issue by Altonss** [eskerda/pybikes#493]
We could check more often this file https://github.com/NABSA/gbfs/blob/master/systems.csv
It is updated often and we could have some runners that check for systems (either not yet implemented, or ones we could replace with gbfs)

**Comment by francoisfds** [comment]
Hello. Checking automaticaly is a good idea, however I think this file contains some networks that have only romaing bikes and no stations (example "Lime").

**Comment by Altonss** [comment]
> Hello. Checking automaticaly is a good idea, however I think this file contains some networks that have only romaing bikes and no stations (example "Lime").

Yes you are right! I noticed the same (some networks also don't exist anymore)
